### ISSUE TITLE
Enum.Tostring() Error when using up arrow then down arrow.

### ISSUE THREAD

**Issue by Malte0621** [aura-systems/Aura-Operating-System#208]
I just compiled the os from source and it seems like i already found a bug.
By using a few random commands then using up arrow and down array after. It seems like it causes a crash with the exception bellow:

![image](https://user-images.githubusercontent.com/50240920/122278870-b129b580-cee7-11eb-831d-923cbe6ff13a.png)

```
Exception:
Parameter name: <Enum.ToString> not implemented
Aura Version: 0.5.1
Aura Revision 160620212104
```

It seems like its just a non-implemented cosmos feature. Ill check if it can be replaced or if the cosmos part can be changed to support it.

**Comment by Malte0621** [comment]
Update: It seems like this line is making an error, i have 0 clue how..
I mean, its all strings. Not enums. .-.
![image](https://user-images.githubusercontent.com/50240920/122285152-a9b9da80-ceee-11eb-9667-729e85c9dfc3.png)
`ConsoleImpl.cs:638`

**Comment by Malte0621** [comment]
Another update: Only seems to trigger when the command `help` is used then using up and down arrow.

**Comment by valentinbreiz** [comment]
Thanks for your submit but this is a known issue (https://github.com/aura-systems/Aura-Operating-System/issues/160) :)

**Comment by Kiirx** [comment]
Could be closed if #160 is fixed

**Comment by valentinbreiz** [comment]
Fixed.
